docs/content/latest/manage/backup-restore/point-in-time-restore-ysql.md

```diff
@@ -0,0 +1,331 @@
+
+1. Verify the snapshot is restored:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                     State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    Restoration UUID                      State
+    bd7e4e52-b763-4b95-87ce-9399e1ac206e  RESTORED
+    ```
+
+1. Verify the data is restored:
+
+    ```sql
+    yugabyte=# select * from employees;
+    ```
+
+    ```output
+    employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1223 | Lucille Ball   | Operations |  70000
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (4 rows)
+    ```
+
+### Restore from a relative time
+
+In addition to restoring to a particular timestamp, you can also restore to a relative time, such as "ten minutes ago". In this example, you'll delete some data from the existing `employees` table, then restore the state of the database to what it was five minutes prior.
+
+1. Wait five minutes after you complete the steps in the previous section. This is so that you can use a known relative time for the restore.
+
+1. Remove employee 1223 from the table:
+
+    ```sql
+    yugabyte=# delete from employees where employee_no=1223;
+
+    yugabyte=# select * from employees;
+    ```
+
+    ```output
+    employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (4 rows)
+    ```
+
+1. Restore the snapshot you created earlier:
+
+    ```sh
+    $ bin/yb-admin restore_snapshot bb5fc435-a2b9-4f3a-a510-0bacc6aebccf "minus 3m"
+    ```
+
+1. Verify the data is restored:
+
+    ```sql
+    yugabyte=# select * from employees;
+    ```
+
+    ```output
+    employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1223 | Lucille Ball   | Operations |  70000
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (4 rows)
+    ```
+
+## Limitations
+
+This is a BETA feature, and is in active development. Currently, you can **restore data only**. The feature doesn't support metadata; in other words, it won't currently roll back operations such as CREATE, ALTER, TRUNCATE, and DROP TABLE.
+
+Development for this feature is tracked in [issue 7120](https://github.com/yugabyte/yugabyte-db/issues/7120). Some forthcoming features include:
+
+* Automatic configuration
+* Early metadata support, such as undoing CREATE TABLE operations
+* More complete metadata support: TRUNCATE, DROP, ALTER
+* Per-database restore for YSQL
```

Comment:
not sure if to list these out like this, since it seems weird to read "early support for X" vs "more support for Y"

if you want to call something out, maybe split into
- support for undoing metadata operations, such as CREATE, ALTER, TRUNCATE or DROP TABLE
- schedules to ensure in-cluster snapshots are taken at user defined intervals
- options to restore at different granularities, such as a complete YSQL database vs the whole YCQL dataset

Reply:
Updated...